src/components/ResultPage.vue

```diff
@@ -26,12 +26,24 @@
         See High Scores
       </h3>
     </router-link>
+    <div class="learning-suggestion">
+      <a v-bind:href="currentLogo.url" target="_blank">
+        <js-logo :size="55"></js-logo>
+      </a>
+      <p>I am <strong>{{currentLogo.name}}!</strong> Click me to learn more.</p>
```

Comment:
Can you set `<strong>{{currentLogo.name}}!</strong>` to be `<a v-bind:href="currentLogo.url" target="_blank"><strong>{{currentLogo.name}}!</strong></a>`. On my initial attempt, I was trying to click it and thought that to be the URL. I think both the logo and the text can be links.

Also checked out that it will require a bit of CSS changes. Can you also make sure that it has 
```
color: #66BB6A;
text-decoration: none;
```